test/integration/builder/transaction.js

```diff
@@ -659,5 +659,46 @@ module.exports = function(knex) {
         return builder;
       });
     });
+
+    it('#3690 does not swallow exception when error during transaction occurs', async function() {
+      const killConnectionMap = {
+        mysql: async (connection) => knex.raw('KILL ?', [connection.threadId]),
```

Comment:
so mysql and mysql2 are same? can we set arrow function to const and reuse it?

Reply:
done